skunkworks_crow/src/main/java/org/odk/share/views/ui/main/MainActivity.java

```diff
@@ -292,4 +294,22 @@ private void setUpLoader() {
             }
         }
     }
+
+    // Add toast "Press BACK to exit"
+    @Override
+    public void onBackPressed() {
+        if(BACK_FLAG){
+            super.onBackPressed();
+            return;
+        }
+        BACK_FLAG = true;
+        Toast.makeText(getApplicationContext(),"Press BACK to exit",Toast.LENGTH_SHORT).show();
+
+        new Handler().postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                BACK_FLAG = false;
+            }
+        },2000);
```

Comment:
If the delay time `2000` is static, is better to use a static variable to present.